common/src/main/java/tds/common/configuration/CacheConfiguration.java

```diff
@@ -37,23 +41,25 @@ public CacheManager cacheManager() {
     }
 
     @Bean
-    @Qualifier(CacheType.SHORT_TERM)
+    public KeyGenerator keyGenerator() {
+        return new CacheKeyGenerator();
+    }
+
+    @Bean
```

Comment:
So you didn't need to name the bean [CacheType.SHORT_TERM](https://github.com/SmarterApp/TDS_Common/blob/develop/common/src/main/java/tds/common/configuration/CacheType.java#L9)?